apps/passport-client/components/screens/AddSubscriptionScreen.tsx

```diff
@@ -463,50 +478,110 @@ function AlreadySubscribed({
           />
         </>
       )}
-      <Spacer h={16} />
-      You subscribed to this feed on{" "}
-      {new Date(existingSubscription.subscribedTimestamp).toLocaleDateString(
-        navigator.language,
-        options
-      )}
       <Spacer h={8} />
-      {!isDefaultSubscription(existingSubscription) && (
-        <Button onClick={onUnsubscribeClick} size="small" style="danger">
+
+      <>
+        You subscribed to this feed on{" "}
+        {new Date(existingSubscription.subscribedTimestamp).toLocaleDateString(
+          navigator.language,
+          options
+        )}
+      </>
+
+      <Spacer h={8} />
+      <FolderExplorerContainer>
+        {folders.map((folder) => (
+          <FolderCard
+            key={folder}
+            onFolderClick={(): void => {
+              goToFolder(folder);
+            }}
+            folder={folder}
+          />
+        ))}
+      </FolderExplorerContainer>
+      <Spacer h={8} />
+      {isDefaultSubscription(existingSubscription) ? (
+        <></>
+      ) : (
+        <Button onClick={onUnsubscribeClick} style="danger">
           Unsubscribe
         </Button>
       )}
     </div>
   );
 }
 
-function FolderLink({ folder }: { folder: string }): JSX.Element {
-  const navigate = useNavigate();
-  const goToFolder = useCallback(() => {
-    navigate(`/?folder=${encodeURIComponent(folder)}`);
-  }, [folder, navigate]);
-  return (
-    <FolderButton>
-      <Button onClick={goToFolder}>{folder}</Button>
-    </FolderButton>
-  );
-}
-
 const InfoRowContainer = styled.div`
-  padding: 16px;
-  border: 1px solid white;
-  border-radius: 12px;
-  background: var(--bg-lite-gray);
+  ${({
+    expanded,
+    lockExpanded
+  }: {
+    expanded;
+    lockExpanded;
+  }): FlattenSimpleInterpolation => css`
+    transition:
+      background-color 100ms,
+      border 100ms;
+
+    ${expanded
+      ? css`
+          border: 1px solid white;
+          padding: 16px;
+          border-radius: 16px;
+          background-color: rgba(255, 255, 255, 0.05);
+          cursor: pointer;
+
```

Comment:
```suggestion
user-select: none;

```

Reply:
i get this when i click

<img width="456" alt="Screenshot 2024-02-15 at 4 35 25 PM" src="https://github.com/proofcarryingdata/zupass/assets/36896271/fa5438c6-793c-4c2b-a454-812ee7ed5bd2">